docs/pages/releases/2021-11-23.mdx

```diff
@@ -0,0 +1,262 @@
+
+Our **Release Candidate** for **General Availability** is finally here! This is our RC for **Keystone 6** with numerous tweaks and fixes across the project. <Emoji symbol="⭐️" alt="Gold star"/>
+
+Next week we're planning to move **Keystone 6** to General Availability with this release, graduating to the `@keystone-6` namespace on npm.
+
+We encourage all Keystone developers to upgrade to this release.
+
+```json
+"keystone-next/keystone": "xx.x.x",
+```
+
+**⚠️ &nbsp; This release contains breaking changes, please see below!**
+
+!> Like this release? [Give us a star](https://github.com/keystonejs/keystone) on GitHub!
+
+## Shorter Relationship Names <Emoji symbol="🤏" alt="Pinching hand" />
+
+x> **⚠️ &nbsp; Breaking change, you may lose data if you do not follow the guide below.**
+
+The names of **one-sided** and **two-sided**, **many-many** relationships has been **shortened**. Two-sided many-many relationship names contain only the left-hand side names now; and the `_many` suffix has been dropped from one-sided many-many relationships.
+
+This reduces the probability that you will exceed [PostgreSQL's 63 character limit for identifiers](https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS) with typical usage.
+
+There are two ways to update:
+
+- Set `db.relationName` on many to many relations, this allows your database to remain unchanged.
+
+- Rename your many relation tables using a migration, updating your DB.
+
+### Set `db.relationName` on many to many relations
+
+Rather than doing a migration, you can set the new field property `db.relationName`, for either side of a many-to-many relationship field.
+
+If set to the existing relation name, your database will remain unchanged.
+
+For example, given a schema like this:
+
+```ts
+Post: list({
+  fields: {
+    tags: relationship({ ref: 'Tag.posts', many: true }),
+  },
+}),
+Tag: list({
+  fields: {
+    posts: relationship({ ref: 'Post.tags', many: true }),
+  },
+}),
+```
+
+Before this release, the generated Prisma schema looked like this:
+
+```prisma
+// This file is automatically generated by Keystone, do not modify it manually.
+// Modify your Keystone config when you want to change this.
+
+datasource postgresql {
+  url      = env("DATABASE_URL")
+  provider = "postgresql"
+}
+
+generator client {
+  provider   = "prisma-client-js"
+  output     = "node_modules/.prisma/client"
+  engineType = "binary"
+}
+
+model Post {
+  id   String @id @default(cuid())
+  tags Tag[]  @relation("Post_tags_Tag_posts")
+}
+
+model Tag {
+  id    String @id @default(cuid())
+  posts Post[] @relation("Post_tags_Tag_posts")
+}
+```
+
+By adding `db: { relationName: 'Post_tags_Tag_posts' }` to one side of the many-to-many relationship; you can preclude yourself from a migration.
+
+**Note:** It doesn't matter which side of the relationship you put this property, but it should be only on one side; otherwise you will receive an error.
```

Comment:
```suggestion
?> It doesn't matter which side of the relationship you put this property, but it should be only on one side; otherwise you will receive an error.
```